app.go

```diff
@@ -114,6 +114,8 @@ type App struct {
 	customMethod bool
```

Comment:
I think we can do the same for customMethod

Reply:
Done